### PR TITLE
Add clear icon for number type columns

### PR DESCRIPTION
**Pull request by nomego**
See #26

Should be merged after #94 

Signed-off-by: Patrik Kullman <patrik.kullman@neovici.se>

### CODE REVIEW

File: cosmoz-omnitable-column-number.html [hunk 1]
```diff
@@ -21,10 +22,14 @@
 				<div class="dropdown-content" slot="dropdown-content" style="padding: 15px; min-width: 100px;">
 					<h3 style="margin: 0;">[[ title ]]</h3>
 					<paper-input type="number" label="[[ _('From', t) ]]" value="{{ _filterInput.min }}"
-						min="[[ _limit.fromMin ]]" max="[[ _limit.fromMax ]]">
+						min="[[  _limit.fromMin ]]" max="[[ _limit.fromMax ]]">
+						<iron-icon style="cursor: pointer;" hidden$="[[ !_filterInput.min.length ]]"
+							suffix slot="suffix" icon="clear" on-click="_clearInput"></iron-icon>
```
Comment (Wurper) [inline cosmoz-omnitable-column-number.html]: _clearInput does not seem to be defined.

File: cosmoz-omnitable-column-number.html [hunk 2]
```diff
@@ -21,10 +22,14 @@
 				<div class="dropdown-content" slot="dropdown-content" style="padding: 15px; min-width: 100px;">
 					<h3 style="margin: 0;">[[ title ]]</h3>
 					<paper-input type="number" label="[[ _('From', t) ]]" value="{{ _filterInput.min }}"
-						min="[[ _limit.fromMin ]]" max="[[ _limit.fromMax ]]">
+						min="[[  _limit.fromMin ]]" max="[[ _limit.fromMax ]]">
```
Comment (Wurper) [inline cosmoz-omnitable-column-number.html]: Extra space. [[  _limit.fromMin ]]

Comment (Wurper) [inline cosmoz-omnitable-column-number.html]: Extra space.

File: cosmoz-omnitable-column-number.html (hunk 1 above)
Comment (Wurper) [inline cosmoz-omnitable-column-number.html]: _clearInput does not seem to be defined.

File: cosmoz-omnitable-column-number.html (hunk 2 above)
Comment (plequang) [reply cosmoz-omnitable-column-number.html]: My bad. The PR needed to be rebased and I rebase using github UI.

File: cosmoz-omnitable-column-number.html (hunk 1 above)
Comment (plequang) [reply cosmoz-omnitable-column-number.html]: Also my bad. Rebasing the PR with master dropped the _clearInput function

**Comment by Wurper** [comment]
* If you reload with values in the url the remove button does not show until a value updated.
* The increase/decrease value button is shifted to the left by the remove icon when a value is set and its easy accidentally click the remove icon right after.

**Comment by plequang** [comment]
Not needed anymore since  https://github.com/Neovici/cosmoz-omnitable/pull/99  ?

**Comment by nomego** [comment]
I still think this adds convenience for the user when they want to clear either from or to value.

**Comment by plequang** [comment]
OK my bad. I thought it was related to the same "clear" button/icon

**Comment by nomego** [comment]
@MicoNeovici Is this needed anymore? Didn't we fix this in a separate PR ?

**Comment by MicoNeovici** [comment]
I´m not sure but I think this is fixed in another PR, however I think the pixel discrepancy between demo and Cosmoz is still there. (Not fixed in this PR either)

**Comment by megheaiulian** [comment]
No longer relevant.

**Pull request closed without merging**